# Pitfall 05: Max and Min Points

## input:

```python
runNRM(
    f = "sin(2*x) - x",
    x0 = 25,
    know_root=2.0945514815423265,
    tol=1e-6,
    max_iter=15
    )
```

result:

```plaintext
✗ Não convergiu. Iterações realizadas: 15
Erro: Número máximo de iterações atingido.
Último valor calculado: 0.9478
Iteração 1: x = 25.00000000000000000, f(x) = -25.26237485370392832
Iteração 2: x = 52.16582858282131951, f(x) = -52.77823387512278686
Iteração 3: x = 31.71777126176566597, f(x) = -31.15008760232964491
Iteração 4: x = 79.90090781702912182, f(x) = -79.49368643227656150
Iteração 5: x = 51.77806476785846712, f(x) = -51.66189959666515819
Iteração 6: x = 34.47935568423210384, f(x) = -34.63504674889668422
Iteração 7: x = 69.98021173104879722, f(x) = -68.99291291439081419
Iteração 8: x = 17.62355448579880601, f(x) = -18.25977523555878790
Iteração 9: x = 10.44318731356559482, f(x) = -9.54982467298074589
Iteração 10: x = 5.41345035095846594, f(x) = -6.39925865669175309
Iteração 11: x = 0.62268894404425179, f(x) = 0.32482808825663678
Iteração 12: x = 1.52351429605588651, f(x) = -1.42909110924945737
Iteração 13: x = 1.04572747308561098, f(x) = -0.17823573649186741
Iteração 14: x = 0.95638191595725841, f(x) = -0.01428521230377788
Iteração 15: x = 0.94783139178224363, f(x) = -0.00013803229089726

Erros Absolutos:
Iteração 1: Erro = 22.90544851845767482
Iteração 2: Erro = 50.07127710127899434
Iteração 3: Erro = 29.62321978022334079
Iteração 4: Erro = 77.80635633548679664
Iteração 5: Erro = 49.68351328631614194
Iteração 6: Erro = 32.38480420268977866
Iteração 7: Erro = 67.88566024950647204
Iteração 8: Erro = 15.52900300425647906
Iteração 9: Erro = 8.34863583202326787
Iteração 10: Erro = 3.31889886941613943
Iteração 11: Erro = 1.47186253749807472
Iteração 12: Erro = 0.57103718548644000
Iteração 13: Erro = 1.04882400845671553
Iteração 14: Erro = 1.13816956558506810
Iteração 15: Erro = 1.14672008976008288
```